app/pods/components/course-library/doubts/template.hbs

```diff
@@ -0,0 +1,10 @@
+<div>
+<Player::PlayerDoubtsList @doubts={{@payload}} />
+</div>
+<style>
+  div > .input-search {
```

Comment:
Also make your rule more specific to this usecase, this will apply this globally to all `.input-search` which are inside a `div`. Keep css as specific as possible.